packages/domutils/tsconfig.json

```diff
@@ -1,12 +1,15 @@
 {
   "compilerOptions": {
     "composite": true,
+    "sourceMap": true,
```

Comment:
Won't we also need something like `"inlineSources": true,`? Otherwise, won't we just run into the same issue as in #41 again?

Back in December I experimented extensively with various combinations of the typescript and webpack source map config options. I found that when using both `"sourceMap"` and `"inlineSources"`, you also end up needing to set `"sourceRoot"`. Otherwise, you run the very real risk of ending up with 15 indistinguishable `index.ts` files next to each other at the root of the inferred filesystem of your bundle (ie what you see in the left panel on the "sources" tab of devtools), which makes debugging a real pain. Ideally you want the source files to all be appropriately sorted in their respective `@lumino/whatever` dirs.

However, I was never able to find the "correct" value to use for `"sourceRoot"`. For one thing, when building Lumino packages as part of Jupyterlab's build, the effect of `"sourceRoot"` seems to change depending on whether you do do a dev mode build or just the regular app build. Do you have any insight into the issue, @GordonSmith?

Reply:
In this case it should be fine as the source maps get bundled as part of the rollup (and minification) processes (including the es6 module which webpack uses).

I will do sanity check on one of the WebPack tests today.

Re the sourceRoot setting - So far I have never had to use this setting in any of my projects...